supervisor/addons/addon.py

```diff
@@ -572,13 +575,15 @@ def _write_tarfile():
                 """Write tar inside loop."""
                 with tar_file as snapshot:
                     # Snapshot system
-                    snapshot.add(temp, arcname=".")
```

Comment:
Revert this, we need it only for data folder

Reply:
You pointed me to this line? I dont really get what you want as the directory is being added inside the `atomic_contents_add` function.

Please take a little more time for the review so you dont waste your and my time. I know this is open source and I appreciate the feedback but its quite annoying if you change your requirements from commit to commit.

Thanks.